easybuild/tools/module_generator.py

```diff
@@ -174,25 +174,6 @@ def get_modules_path(self, fake=False, mod_path_suffix=None):
 
         return os.path.join(mod_path, mod_path_suffix)
 
-    def prepare(self, fake=False):
```

Comment:
I haven't checked, but I assume this method is not used anywhere else, including easyblocks, right?